Gems/ScriptCanvas/Code/Editor/Components/EditorUtils.cpp

```diff
@@ -31,6 +31,8 @@ AZ_POP_DISABLE_WARNING
 
 namespace ScriptCanvasEditor
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether